label.py

```diff
@@ -30,10 +32,21 @@ def get_new_image():
         return
     current_image = images_to_show.pop()
     print("data/%s_firefox.png" % current_image)
-    img = ImageTk.PhotoImage(Image.open("data/%s_firefox.png" % current_image))
+
+    img1 = Image.open("data/%s_firefox.png" % current_image)
```

Comment:
Can you rename the variables to make it clearer what they are? There are too many `img` now :)
E.g. replace `img1` with `img_firefox`, `img2` with `img_firefox`, and so on.